translations/ga/census_individual_gb_nir_ga.po

```diff
@@ -2,3312 +2,4784 @@ msgid ""
+msgctxt "Main job"
+msgid "Their circumstances may have changed during the coronavirus pandemic. Answer all questions based on their situation as it is now."
+msgstr "D'fhéadfadh sé gur tháinig athrú ar a dtosca le linn na paindéime Coróinvíris. Freagair gach ceist bunaithe ar an dóigh atá ar chúrsaí faoi láthair."
+
+#. Content page description
+msgctxt "Last main job"
+msgid "The next set of questions is about your last main job. This is the most recent job you had."
+msgstr "Tá an chéad tacar eile de cheisteanna faoin phríomhphost dheireanach a bhí agat. Is é seo an post is déanaí a bhí agat."
+
+#. Content page description
+msgctxt "Last main job"
+msgid "If you had more than one job at the same time, answer for the job in which you usually worked the most hours."
+msgstr "Má bhí níos mó ná post amháin agat ag an am céanna, freagair faoin phost inar oibrigh tú an méid is mó uaireanta de ghnáth."
+
+#. Content page description
+msgctxt "Last main job"
+msgid "The next set of questions is about <em>{person_name_possessive}</em> last main job. This is the most recent job they had."
+msgstr "Tá an chéad tacar eile de cheisteanna faoi phríomhphost deireanach <em>{person_name_possessive}</em>. Is é seo a post is déanaí a bhí acu."
+
+#. Content page description
+msgctxt "Last main job"
+msgid "If they had more than one job at the same time, answer for the job in which they usually worked the most hours."
+msgstr "Má bhí níos mó ná post amháin acu ag an am céanna, freagair faoin phost inar oibrigh siad an méid is mó uaireanta de ghnáth."
+
+#. Question text
+msgid "What type of accommodation is <em>{household_address}</em>?"
+msgstr "Cén sórt cóiríochta é <em>{household_address}</em>?"
+
+#. Question text
 msgid "Are you answering these questions for yourself or for someone else?"
 msgstr "An bhfuil tú ag freagairt na gceisteanna seo ar do shon féin nó ar son duine éigin eile?"
 
+#. Question text
 msgid "What is your name?"
 msgstr "Cad é an t-ainm atá ort?"
 
+#. Question text
 msgid "What is their name?"
 msgstr "Cad é an t-ainm atá orthu?"
 
+#. Question text
 msgid "What is your position in this establishment?"
 msgstr "Cad é an post atá agat san áit seo?"
 
-msgid "What is an establishment?"
-msgstr "Cad í an rud í áit?"
+#. Question text
+msgid "What is <em>{person_name_possessive}</em> position in this establishment?"
+msgstr "Cad í an fhreagracht atá ag <em>{person_name_possessive}</em> san áit seo?"
 
+#. Question text
 msgid "What is your date of birth?"
 msgstr "Cad é do dháta breithe?"
 
+#. Question text
+msgid "What is <em>{person_name_possessive}</em> date of birth?"
+msgstr "Cad é dáta breithe <em>{person_name_possessive}</em>?"
+
+#. Question text
+msgid "You are {age} old. Is this correct?"
+msgstr "Tá tú {age} d'aois. An bhfuil sé seo ceart?"
+
+#. Question text
+msgid "<em>{person_name}</em> is {age} old. Is this correct?"
+msgstr "Tá <em>{person_name}</em> {age} bliain d'aois. An bhfuil sé seo ceart?"
```

Comment:
Same as the query on the household schema, I think "bliain" needs to be removed.

Reply:
changed